incubating/helmfile/step.yaml

```diff
@@ -0,0 +1,236 @@
+                "type": "string",
+                "description": "String of command options and arguments for, helmfile repos - sync repositories from state file (helm repo add && helm repo update)",
+                "default": ""
+            },
+            "CHARTS_PS": {
+                "type": "string",
+                "description": "String of command options and arguments for, helmfile charts - DEPRECATED: sync releases from state file (helm upgrade --install)",
+                "default": ""
+            },
+            "DIFF_PS": {
+                "type": "string",
+                "description": "String of command options and arguments for, helmfile diff - diff releases from state file against env (helm diff)",
+                "default": ""
+            },
+            "TEMPLATE_PS": {
+                "type": "string",
+                "description": "String of command options and arguments for, helmfile template - template releases from state file against env (helm template)",
+                "default": ""
+            },
+            "LINT_PS": {
+                "type": "string",
+                "description": "String of command options and arguments for, helmfile lint - lint charts from state file (helm lint)",
+                "default": ""
+            },
+            "SYNC_PS": {
+                "type": "string",
+                "description": "String of command options and arguments for, helmfile sync - sync all resources from state file (repos, releases and chart deps)",
+                "default": ""
+            },
+            "APPLY_PS": {
+                "type": "string",
+                "description": "String of command options and arguments for, helmfile apply - apply all resources from state file only when there are changes",
+                "default": ""
+            },
+            "STATUS_PS": {
+                "type": "string",
+                "description": "String of command options and arguments for, helmfile status - retrieve status of releases in state file",
+                "default": ""
+            },
+            "DELETE_PS": {
+                "type": "string",
+                "description": "String of command options and arguments for, helmfile delete - DEPRECATED: delete releases from state file (helm delete)",
+                "default": ""
+            },
+            "DESTROY_PS": {
+                "type": "string",
+                "description": "String of command options and arguments for, helmfile destroy - deletes and then purges releases",
+                "default": ""
+            },
+            "TEST_PS": {
+                "type": "string",
+                "description": "String of command options and arguments for, helmfile test - test releases from state file (helm test)",
+                "default": ""
+            }
+        }
+    }
+  steps:
+    main:
+      name: helmfile
+      image: 'dustinvanbuskirk/cfstep-helmfile:${{HELMFILE_VERSION}}'
```

Comment:
It is better to put the image under codefresh registry (https://hub.docker.com/r/codefresh)

Reply:
Completed